app/views/layouts/admin.html.haml

```diff
@@ -3,15 +3,15 @@
   = dashboard_stylesheet_tag("surveys")
 
 - content_for :javascripts do
-  /
+  = hot_javascript_tag("main")
 
 !!!
 %html{ dir: rtl? ? "rtl" : nil, class: body_class(request)}
   = render "shared/head"
   %body{class: body_class(request)}
     = render "shared/flash"
     %div.wrapper
-      = render "shared/nav"
+      #nav_root{"data-rooturl" => main_app.root_url, "data-logopath" => logo_path, "data-fluid" => "false", "data-exploreurl" =>  main_app.explore_path, "data-explorename" => t(Features.wiki_ed? ? "application.explore" : "courses_generic.explore"), "data-usersignedin" => user_signed_in? ? 'true' : 'false', "data-ifadmin" => current_user&.admin? ? 'true' :'false', "data-trainingurl": main_app.training_url, "data-disable_training": Features.disable_training? ? 'true' : 'false', "data-help_disabled" => Features.disable_help? ? 'true' : 'false', "data-ask_url" => main_app.ask_url, "data-user_permissions" => permissions, "data-wiki_ed" => Features.wiki_ed? ? 'true' : 'false', "data-language_switcher_enabled" => language_switcher_enabled, "data-username" => current_user&.username, "data-destroyurl" => main_app.destroy_user_session_url, "data-omniauth_url" => main_app.user_mediawiki_omniauth_authorize_url}
```

Comment:
In all these menus where you do something like `Features.wiki_ed? ? 'true' : 'false'`, you could just do `Features.wiki_ed?.to_s` instead.